src/test/java/sut/BranchAndLinearCoverageTesting/TSTgetTest.java

```diff
@@ -32,14 +30,12 @@ public void testWithLenghtZero() {
 		});	
 	}
 	
-	//TODO -> duvido que seja boa pratica fazer assim
 	@Test
-	public void testWithGetNull() {
+	public void testReturnNullIfKeyNotPresent() {
 		String input = "test";
 		assertNull(new TST<String>().get(input));
```

Comment:
inline the key. For even more clarity, you can use sth like "notPresentInTrieKey"